infra/helm-charts/mongo/charts/ingestbackup/src/restore.sh

```diff
@@ -31,4 +31,7 @@ else
 fi
 
 mongo --host=$MONGO_HOST admin -u $MONGO_USER -p $MONGO_PASSWORD --eval 'db.shutdownServer()'
+
+echo "Successful run!"
```

Comment:
Should we add some summary stats for the run? For example: time, duration, amount of records for each collection, etc.?

Reply:
That's a good suggestion, I'll create that in a separate PR.

Reply:
Also, do you have suggestions on what checks to add? Currently, the verify script is only checking if there are 100 submissions.

Reply:
What checks do you think is helpful?

I was thinking of getting the difference in the size of the backup files. If it shrunk a lot or what, then something may be wrong with the backup file... 🤔

Reply:
>time, duration, amount of records for each collection, etc.?

we could also add that in the info alert! :D

Reply:
will merge now but i'll do another PR for the improvement of verify 'checks/info/stats'